src/components/pages/Categories/Category.vue

```diff
@@ -41,11 +49,27 @@ export default {
     errorMessage() {
       return this.$store.state.categories.errorMessage;
     },
+    doneMessage() {
+      return this.$store.state.categories.doneMessage;
+    },
   },
 
   created() {
     this.$store.dispatch('categories/getAllCategories');
   },
+
+  methods: {
+    updateValue($event) {
```

Comment:
[imo]

引数名には、Vue で使えるプロパティ名と区別するため、`$` は付与しないのが一般的かと思います。

```suggestion
    updateValue(event) {
```

Reply:
コミットa6bfce14004d28f9cc9504ed138c74f729b10e47にて修正しました。